### ISSUE TITLE
A HiDPI related patch

### ISSUE THREAD

**Issue by lextm** [dockpanelsuite/dockpanelsuite#320]
https://github.com/nabeshin/dockpanelsuite/commits/hotfix/hidpi

**Comment by lextm** [comment]
Closed via b6c730a81d54d673c570f6b0f26ae8916dd41eba

**Comment by lextm** [comment]
The recent work on color palette based theme seems to give me better insight into the area. However, I am not yet familiar with WinForms under HiDPI. So will do more experiments and come back.

**Comment by lextm** [comment]
I just tested on the following environment,

* Windows 10 with latest patches
* .NET Framework 4.6.1
* Latest build of the Sample project from master
* DPI set to 125%

Due to the default DPI scaling in WinForms, there is significant blurring in the final screen shot, which makes the button positions difficult to measure. But compared to VS WPF screen shots, the offsets can be only one or two pixels.

As differences is almost negligible, will not pursue on this issue, as well as other DPI issues.

There were changes I made in the past few days on button positions for DPI 100% case, which matches the relevant VS themes. Those won't be rolled back.
